sbt

```diff
@@ -26,6 +26,9 @@ declare -r sbt_launch_mvn_snapshot_repo="http://repo.scala-sbt.org/scalasbt/mave
 declare -r default_jvm_opts_common="-Xms512m -Xss2m"
 declare -r noshare_opts="-Dsbt.global.base=project/.sbtboot -Dsbt.boot.directory=project/.boot -Dsbt.ivy.home=project/.ivy"
 
+declare -r default_coursier_launcher_version="1.2.14"
+declare coursier_launcher_version
```

Comment:
Let's default this to `""` here and drop the `else` on `SBTX_COURSIER`

Reply:
Done.